libafl/src/statistics/user_stats/mod.rs

```diff
@@ -59,11 +59,14 @@ pub enum AggregatorOps {
 }
 
 /// Aggregate user statistics according to their ops
-pub(super) fn aggregate_user_stats(client_stats_manager: &mut ClientStatsManager, name: &str) {
+pub(super) fn aggregate_user_stats(
+    client_stats_manager: &mut ClientStatsManager,
+    name: Cow<'static, str>,
+) {
     let mut gather = client_stats_manager
         .client_stats()
         .iter()
-        .filter_map(|client| client.user_stats.get(name));
+        .filter_map(|client| client.user_stats.get(name.as_ref()));
```

Comment:
This gets the name as `ref` -> if we pass `&Cow` around we would have less clones, I think

Reply:
There's a lot of early returns where you don't have to clone

Reply:
Hmmm, ok. I'll update it.